package/MDAnalysis/analysis/align.py

```diff
@@ -380,6 +382,177 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
+            self.weights = np.asarray(ref_atoms.masses)
+        else:
+            self.weights = None
+
+        if os.path.exists(self.filename) and not force:
+            logger.warn("{0} already exists and will NOT be overwritten; use force=True if you want this".format(self.filename))
+
+        logger.info("RMS-fitting on {0:d} atoms.".format(len(self.ref_atoms)))
+
+        self.quiet = quiet
+        if self.quiet:
+            # should be part of a try ... finally to guarantee restoring the log level
+            logging.disable(logging.WARN)
+
+        self._setup_frames(frames, start,stop,step)
+
+    def _prepare(self):
+        # reference centre of mass system
+        self.ref_com = self.ref_atoms.center_of_mass()
+        self.ref_coordinates = self.ref_atoms.positions - self.ref_com
+
+        # allocate the array for selection atom coords
+        self.mobile_coordinates = self.mobile_atoms.positions.copy()
+        self.rmsd = np.zeros(())
+
+        # R: rotation matrix that aligns r-r_com, x~-x~com
+        #    (x~: selected coordinates, x: all coordinates)
+        # Final transformed traj coordinates: x' = (x-x~_com)*R + ref_com
+        self.rot = np.zeros(9, dtype=np.float64)  # allocate space for calculation
+        self.R = np.matrix(self.rot.reshape(3, 3))
+
+    def _single_frame(self):
+
+        self.mobile_com = self.mobile_atoms.center_of_mass().astype(np.float32)
+        self.mobile_coordinates[:] = self.mobile_atoms.positions - self.mobile_com
+
+        # Need to transpose coordinates such that the coordinate array is
+        # 3xN instead of Nx3. Also qcp requires that the dtype be float64
+        # (I think we swapped the position of ref and traj in CalcRMSDRotationalMatrix
+        # so that R acts **to the left** and can be broadcasted; we're saving
+        # one transpose. [orbeckst])
+        np.append(self.rmsd, qcp.CalcRMSDRotationalMatrix(self.ref_coordinates.T.astype(np.float64),
+                self.mobile_coordinates.T.astype(np.float64), self.natoms, self.rot, self.weights))
+
+        self.R[:, :] = self.rot.reshape(3, 3)
+
+        # Transform each atom in the trajectory (use inplace ops to avoid copying arrays)
+        # (Marginally (~3%) faster than "ts.positions[:] = (ts.positions - x_com) * R + ref_com".)
+        self._ts.positions -= self.mobile_com
+        self._ts.positions[:] = self._ts.positions * self.R  # R acts to the left & is broadcasted N times.
+        self._ts.positions += self.ref_com
+
+        self.writer.write(self.mobile.atoms)  # write whole input trajectory system
+
+
+    def _conclude(self):
+        self.writer.close()
+        if self.quiet:
+            logging.disable(logging.NOTSET)
+
+    def save(self, rmsdfile):
+        np.savetxt(rmsdfile, self.rmsd)
+        logger.info("Wrote RMSD timeseries  to file %r", rmsdfile)
+
 def rms_fit_trj(traj, reference, select='all', filename=None, rmsdfile=None, prefix='rmsfit_',
```

Comment:
with a deprecated decorator. Then makes it much easier to see in the code. Look at atomgroup.py for examples.